docs/source/examples/performance_modeling_output.txt

```diff
@@ -94,14 +94,26 @@ Tightest Constraints
        | Aircraft.Wing
  +0.43 : .W >= S·.rho
 
-All Loose Constraints
----------------------
-No constraints had a sensitivity below +1e-05.
+Constraints with sensitivity below +1e-05
+-----------------------------------------
+(none)
 
 Solution difference for selected variables
 ==========================================
 (positive means the argument is smaller)
 
+Constraint differences
+**********************
+
+--- removed in argument
++++ added in argument
+@@ -41,4 +41,3 @@
+          c = (S/A)^0.5
+        "parametrization of wing weight":
+          Aircraft.Wing.W >= S·Aircraft.Wing.rho
+- Wburn[:] >= 0.2·D[:]
+**********************
+
```

Comment:
@1ozturkbe @pgkirsch check out the above

Reply:
Nice! In my current use case, I think you are sort of duplicating the purpose of git, but still a cool idea.

Reply:
Ha, it's true. The main use of this will probably be to assure users that the constraints aren't what caused the changes.